ui/src/app/cron-workflows/cron-workflow-details.scss

```diff
@@ -1,14 +1,14 @@
-@import 'node_modules/argo-ui/src/styles/config';
-
-.workflows-cron-list .workflows-list {
-  &__status {
-    max-width: 80px;
-    display: flex;
-    align-items: center;
-
-    /* checkboxes are not visible and are unused on this page */
-    &--checkbox {
-      appearance: none;
-    }
-  }
-}
+@import 'node_modules/argo-ui/src/styles/config';
+
+.workflows-cron-list .workflows-list {
+  &__status {
+    max-width: 80px;
+    display: flex;
+    align-items: center;
+
+    /* checkboxes are not visible and are unused on this page */
+    &--checkbox {
+      appearance: none;
+    }
+  }
+}
```

Comment:
this file got automatically "modified" because it had inconsistent line endings (i.e. LF vs CRLF)